mmseg/apis/test.py

```diff
@@ -61,6 +62,9 @@ def single_gpu_test(model,
             Mutually exclusive with pre_eval and efficient_test.
             Default: False.
         format_args (dict): The args for format_results. Default: {}.
+        return_all (bool): Return logits value of prediction.
+            Default: False.
+
```

Comment:
I think it would be better to add a kwargs `metric`,
and send `metric` to `pre_eval`,and `pre_eval` call metric function based on `metric`, but not `pre_eval` just call 'iou'. And for different metric function, just fetch different value from `result`